fuse-console-cluster-k8s.yml

```diff
@@ -0,0 +1,620 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: fuse-console-config
+data:
+  hawtconfig.json: |
+    {
+      "about": {
+        "title": "Red Hat Fuse Console",
+        "productInfo": [],
+        "additionalInfo": "The Red Hat Fuse Console eases the discovery and management of Fuse applications deployed on Kubernetes/OpenShift.",
+        "copyright": "",
+        "imgSrc": "../online/img/Logo-RedHat-A-Reverse-RGB.png"
+      },
+      "branding": {
+        "appName": "Fuse Console",
+        "appLogoUrl": "../online/img/Logo-Red_Hat-Fuse-A-Reverse-RGB.png"
+      },
+      "disabledRoutes": [
+        "/camel/source",
+        "/diagnostics",
+        "/jvm/discover",
+        "/jvm/local"
+      ]
+    }
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: fuse-console-service
+  labels:
+    component: fuse-console
+    group: console
+    app: fuse-console
+    version: "1.11"
+spec:
+  type: NodePort
```

Comment:
Should we recommend to use Ingress instead of NodePort?

Reply:
What I think is that because unlike OpenShift what K8s distros offer might differ depending on vendors, it's better to keep the resources as minimal as possible and let users choose the options: NodePort, LoadBalancer, or Ingress. NodePort is the most minimal but still works out of the box for Minikube for instance.

Maybe should we add a note in README that users should choose your preferred way to expose the console on K8s?

Reply:
Right, adding a note in the README is a good idea. Often K8s distros come with a default Ingress controller, so I think it would be worth mentioning Ingress, maybe with a small example.

Reply:
Added a note on other options including Ingress.